### ISSUE TITLE
NOTE: For future cordova builds

### ISSUE THREAD

**Issue by Samleo8** [Samleo8/SisyphusSheep#14]
While building the app for cordova, I encountered the following errors:
```
D8: Program type already present: android.support.v4.os.ResultReceiver$1

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:transformDexArchiveWithExternalLibsDexMergerForDebug'.
> com.android.builder.dexing.DexArchiveMergerException: Error while merging dex archives: 
  Program type already present: android.support.v4.os.ResultReceiver$1
  Learn how to resolve the issue at https://developer.android.com/studio/build/dependencies#duplicate_classes.
```

Apparently, this is due to some plugins using `android-x` and others not. 

After a LOT of searching, I finally found the solution by @dpa99c:
https://github.com/dpa99c/cordova-plugin-androidx-adapter
https://github.com/dpa99c/cordova-plugin-androidx

Thank you, your plugin is an absolute lifesaver!

**tl;dr: Fix:**
```
cordova plugin add cordova-plugin-androidx
cordova plugin add cordova-plugin-androidx-adapter
```
